src/components/AQIGauge/AQIDial.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {Text, Box, Tag, Link, Center} from '@chakra-ui/react';
+import GaugeSVG from './GaugeSVG';
+/**
+ * Interface for the props of the dial
+ * - currentReading is the AQI value that the dial should display
```

Comment:
nit: add tick marks around  `currentReading` and maybe use the variable name `currentAQI`